ingestion/functions/parsing/india/india.py

```diff
@@ -1,44 +1,17 @@
-import boto3
 import json
 import os
-import requests
-import google.auth.transport.requests
-from google.oauth2 import service_account
+import sys
 from datetime import date, datetime
 
-
-LOCAL_DATA_FILE = "/tmp/data.json"
-METADATA_BUCKET = "epid-ingestion"
-SERVICE_ACCOUNT_CRED_FILE = "covid-19-map-277002-0943eeb6776b.json"
-SOURCE_URL_FIELD = "sourceUrl"
-S3_BUCKET_FIELD = "s3Bucket"
-S3_KEY_FIELD = "s3Key"
-S3_KEY_PATH_SEPERATOR = "/"
-
-s3_client = boto3.client("s3")
-
-
-def extract_event_fields(event):
-    if any(
-            field not in event
-            for field in [SOURCE_URL_FIELD, S3_BUCKET_FIELD, S3_KEY_FIELD]):
-        error_message = (
-            f"Required fields {SOURCE_URL_FIELD}; {S3_BUCKET_FIELD}; "
-            f"{S3_KEY_FIELD} not found in input event json.")
-        print(error_message)
-        raise ValueError(error_message)
-    return event[SOURCE_URL_FIELD], event[S3_BUCKET_FIELD], event[S3_KEY_FIELD]
-
-
-def retrieve_raw_data_file(s3_bucket, s3_key):
-    try:
-        local_data_file = LOCAL_DATA_FILE
-        print(f"Retrieving raw data from s3://{s3_bucket}/{s3_key}")
-        s3_client.download_file(s3_bucket, s3_key, local_data_file)
-        return local_data_file
-    except Exception as e:
-        print(e)
-        raise e
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+if ('lambda' not in sys.argv[0]):
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+import parsing_lib
```

Comment:
Like I said, I have other problems getting this to work so I haven't tried the following change. You should be able to import a relative file (which should work both locally and on AWS) with `import ..parsing.parsing_lib`, unless there's something about the AWS environment that doesn't allow that.

Reply:
Ignore this, I managed to fix my venv (actually I don't know what I did, it just started working AFAICT) and relative imports can't be above the top-level module in the filesystem.

Reply:
Yep, ran into that error as well (I believe due to Python file "meta" naming -- as in the argv value -- when running a file directly via `__main__`, as is done with pytest).